compiler/src/main/scala/org/scalajs/core/compiler/PrepJSExports.scala

```diff
@@ -26,6 +26,7 @@ trait PrepJSExports { this: PrepJSInterop =>
       pos: Position,
       isNamed: Boolean,
       isTopLevel: Boolean,
+      isStatic: Boolean,
```

Comment:
Done. There is an ADT/enum `ExportDestination` with 3 values: `Normal`, `TopLevel` and `Static`.